en/Obsidian/Community code of conduct.md

```diff
@@ -2,24 +2,40 @@
 
 These are the behaviors we do not tolerate.
 
-### Major offenses
+### Serious offenses
+
+Once verified, these offenses will result in an immediate ban from our community.
```

Comment:
hmm not necessarily a channel, could be the forum as well

Reply:
I think this means "communication channels," not "discord channels"

Reply:
ok, in that case I think we also need to be clear about what the channels are, so that the reader knows the scope.

The channels I have in mind are:
- Discord
- Forum (English and Chinese)
- Reddit

Reply:
The conversation below between @marcusolsson and @ericaxu is also relevant here:
> > Just a nitpick word of caution when referring to "our community". While a company can act as leaders or stewards within a community, I don't believe any company can own a community.
> 
> I meant it more like in the sense of "our friend", in which case clearly our friend is not owned by us. Is the suggestion to swap out "our community" with "Obsidian's community" or "the community" or "the Obsidian community"?

"from the official Obsidian community" would also cover it without us having to keep track of what exactly what we call "official." I don't think it's worth our trouble to try to keep an up-to-date itemized list in the docs.